CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+### v0.6.2
+
+Locksmith now ignores D-Bus signals that are `nil` or have fields of incorrect
+type, when waiting for a reboot signal from update_engine. This avoids a `panic`
```

Comment:
This should be something that makes sense to end-users who are reading this without prior knowledge about locksmith internals or code structure. I would stick to something simpler like:
```
Locksmith now internally handles DBus signals in a safer way, with additional checks on unexpected/malformed ones. This avoids a panic in the daemon component, which was frequently occurring on shutdown.
```

(I'm not a native speaker, so feel free to reword as you prefer)

Reply:
Makes sense, thanks! Will make this revision.

Reply:
Change committed - wording conveys what is needed.